package.json

```
@@ -21,16 +21,19 @@
     "parser",
     "gfm"
   ],
+  "engines": {
+    "node": ">=15"
+  },
   "author": "Richard Robinson",
   "license": "ISC",
   "dependencies": {
-    "@notionhq/client": "^0.1.9",
+    "@notionhq/client": "^0.4.4",
```

```suggestion
    "@notionhq/client": "^0.4.5",
```
Just grabbing the latest here